src/Admin/MaintainAuthUsers/components/AuthUserSearchResults.js

```diff
@@ -15,6 +17,20 @@ const AuthUserSearchResults = props => {
       <Table.Cell>{a.email}</Table.Cell>
       <Table.Cell>{a.locked ? 'Yes' : 'No'}</Table.Cell>
       <Table.Cell>{a.enabled ? 'Yes' : 'No'}</Table.Cell>
+      <Table.Cell>
+        <Button
+          buttonColour={GREY_3}
+          buttonTextColour={BLACK}
+          mb={0}
+          id={`edit-button-${a.username}`}
+          value={index}
+          onClick={event => {
```

Comment:
You might be able to just use: onClick={handleEdit}

Reply:
agreed, spotted that and ready to fix in my next PR :grin: